### ISSUE TITLE
#[display("{:?}")] could be better

### ISSUE THREAD

**Issue by KalitaAlexey** [frozenlib/parse-display#35]
```rust
#[derive(Display)]
#[display("{:?}")]
enum E {
    A,
    B,
}
```
It generates the following `Display::fmt`:
```rust
match self {
    &Self::A=> f.write_fmt(format_args!("{0:?}", "A")),
    &Self::B=> f.write_fmt(format_args!("{0:?}", "B")),
}
```
I think "{:?}" should generate the following:
```rust
<Self as Debug>::fmt(self, f)
```

**Comment by frozenlib** [comment]
I implemented this specification with de8a9431787053078ed706fc6ff53126c35a4acf.

There are some inefficiencies in the generated code, which I will eventually fix.

**Comment by KalitaAlexey** [comment]
Thank you.
